src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -2151,6 +2156,60 @@ address StubGenerator::base64_vbmi_join_2_3_addr() {
   return start;
 }
 
+address StubGenerator::base64_AVX2_tables_addr() {
```

Comment:
A casual reader might wonder why there's 3 other avx2-tables and then this, so for readability it'd be nice to add "decode" to the name of this new table.